Update governance and maintainers file (#49578)
moby/moby: project/GOVERNANCE.md

# Moby project governance

Moby is the open-source project used to build Docker, jointly maintained by Docker
and the community. As a [community project](https://www.docker.com/community/open-source/),
we abide by a [Code of Conduct](https://github.com/docker/code-of-conduct)
and define a governance which ensures the project is community driven and open to
anyone to contribute.

> Contact moby@docker.com with any questions/concerns about the project or governance.

Reviewer: ```suggestion
Contact moby@docker.com with any questions/concerns about the project governance.
```

Reviewer: Or maybe
```suggestion
Contact moby@docker.com with any questions/concerns about the enforcement of the Code of Conduct
```
assuming most questions can be just asked in public in GitHub Discussions

or the reliability of a release process. But those things distinguish a good
project from a great one.

## Reviewers

A reviewer is a maintainer within the project.
They share in reviewing issues and pull requests and their LGTM counts towards the

> required LGTM count to merge a code change into the project.

Reviewer: Do we need to explicitly define the "required LGTM count"?

Author: I'd leave that to the committers to decide in a follow up. You know my position on that has been to leave the proper number up to the discretion of the merging committer. A set value could be nice to put in branch protections though.
